fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerMultiTargetTest.java

```diff
@@ -867,4 +866,68 @@ void testMultiTargetIndexingBlockerExpiration() {
         // validate
         assertAllValidated(indexes);
     }
+
+    void snooze(int millis) {
+        try {
+            Thread.sleep(millis);
+        } catch (InterruptedException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Test
+    void testForbidConversionOfActiveMultiTarget() throws InterruptedException {
+        // Do not let a conversion of few indexes of an active multi-target session
+        final int numRecords = 59;
+
+        List<Index> indexes = new ArrayList<>();
+        indexes.add(new Index("indexD", new GroupingKeyExpression(EmptyKeyExpression.EMPTY, 0), IndexTypes.COUNT));
+        indexes.add(new Index("indexA", field("num_value_2"), EmptyKeyExpression.EMPTY, IndexTypes.VALUE, IndexOptions.UNIQUE_OPTIONS));
+        indexes.add(new Index("indexB", field("num_value_3_indexed"), IndexTypes.VALUE));
+        indexes.add(new Index("indexC", field("num_value_unique"), EmptyKeyExpression.EMPTY, IndexTypes.VALUE, IndexOptions.UNIQUE_OPTIONS));
+
+        populateData(numRecords);
+
+        FDBRecordStoreTestBase.RecordMetaDataHook hook = allIndexesHook(indexes);
+        openSimpleMetaData(hook);
+        disableAll(indexes);
+
+        Semaphore sem = new Semaphore(1);
+        sem.acquire();
+        Thread t1 = new Thread(() -> {
+            // build index and pause halfway, allowing an active session test
+            try (OnlineIndexer indexBuilder = newIndexerBuilder(indexes)
+                    .setLeaseLengthMillis(TimeUnit.SECONDS.toMillis(20))
+                    .setLimit(4)
+                    .setConfigLoader(old -> {
+                        try {
+                            sem.acquire(); // pause until released
+                        } catch (InterruptedException e) {
+                            throw new RuntimeException(e);
+                        }
+                        sem.release();
+                        return old;
+                    })
+                    .build()) {
+                indexBuilder.buildIndex();
+            }
+        });
+        t1.start();
+        snooze(200); // let the other thread kick in.
+        // Try one index at a time
+        for (Index index : indexes) {
+            try (OnlineIndexer indexBuilder = newIndexerBuilder()
+                    .setIndex(index)
+                    .setIndexingPolicy(OnlineIndexer.IndexingPolicy.newBuilder()
+                            .allowTakeoverContinue(List.of(OnlineIndexer.IndexingPolicy.TakeroverTypes.MULTI_TARGET_TO_SINGLE)))
```

Comment:
Relatedly, would it be worthwhile to add another index not in the build set, and build it here without allowing takeover?

Reply:
I'm not sure that I follow - add it as a multi target? Multi-target subsets takeover are not supported.

Reply:
The test has 4 indexes (`indexA`->`indexD`), would it be worth having another index (`indexE`) that is not included in the previous multi-target index build, and thus is free to be built right now.

Reply:
As each index is being built individually (during the "one at a time" phase), I don't see how it will add any coverage to the other tests in this module.